__tests__/cmds/openapi/index.test.ts

```diff
@@ -328,6 +328,46 @@ describe('rdme openapi', () => {
       return mock.done();
     });
 
+    it('should return spec create info for dry run', async () => {
+      const registryUUID = getRandomRegistryId();
+
+      const mock = getAPIMock()
+        .get(`/api/v1/version/${version}`)
+        .basicAuth({ user: key })
+        .reply(200, { version })
+        .post('/api/v1/api-registry', body => body.match('form-data; name="spec"'))
+        .reply(201, { registryUUID, spec: { openapi: '3.0.0' } });
+
+      const mockWithHeader = getAPIMockWithVersionHeader(version)
+        .get('/api/v1/api-specification')
+        .basicAuth({ user: key })
+        .reply(200, [])
+        .post('/api/v1/api-specification', { registryUUID })
+        .delayConnection(1000)
+        .basicAuth({ user: key })
+        .reply(201, { _id: 1 }, { location: exampleRefLocation });
+
+      const spec = 'petstore.json';
+
+      await expect(
+        openapi.run({
+          key,
+          version,
+          dryRun: true,
+          workingDirectory: './__tests__/__fixtures__/relative-ref-oas',
+        })
+      ).resolves.toBe('🎭 dry run! The spec at petstore.json will be uploaded to version 1.0.0');
+
+      const output = getCommandOutput();
+      expect(output).toBe(
+        chalk.yellow(
+          `⚠️  Warning! 🎭 dry run! This will not create or update any API definition and is used only for debug purposes!
+
+ℹ️  We found ${spec} and are attempting to upload it.`
```

Comment:
@kanadgupta any ideas on how to make this look a little nicer? this expect is kinda ugly right now 😂

Reply:
Instead of using [`toBe()`](https://jestjs.io/docs/expect#tobevalue) (which is an exact match), you could use [`toMatch()`](https://jestjs.io/docs/expect#tomatchregexp--string) (which is a partial match)! See here for an example: https://github.com/readmeio/rdme/blob/5e4722bc038df93be9f8462b6f080e72e79504e2/__tests__/cmds/openapi/index.test.ts#L588-L589